### ISSUE TITLE
OrthogonalTiledMapRenderer does not render TiledMapTileMapObject.

### ISSUE THREAD

**Issue by nathanielbrown** [libgdx/libgdx#4745]
Please ensure you have given all the following requested information in your report.

#### Issue details
OrthogonalTiledMapRenderer does not render TiledMapTileMapObject. The test in the test suite does render them.

I'm not sure if it should be part of the base Renderer or a seperate one. Here is my "easy" solution.

#### Reproduction steps/code
Make a tiled map with a with objects instead of tiles.

https://www.youtube.com/watch?v=ORhC48773bw 

Load the map using the code from the tutorial. https://github.com/libgdx/libgdx/wiki/Tile-maps

#### Version of LibGDX and/or relevant dependencies
1.9.5

#### Stacktrace
N/A

#### Please select the affected platforms
- [ X ] Android
- [ X ] iOS (robovm)
- [ X ] iOS (MOE)
- [ X ] HTML/GWT
- [ X ] Windows
- [ X ] Linux
- [ X ] MacOS

[ObjectTiledMapRenderer.zip](https://github.com/libgdx/libgdx/files/1004406/ObjectTiledMapRenderer.zip)

**Comment by jojorabbit** [comment]
Hi,
Method to render tiled objects is empty in MapRenderer if you dive into source you will see it.
Am not sure if there is need to render those objects, usually they are used for collision so you can easily process layer and render objects by yourself. I have seen many developers just rendering those shapes as debug shapes. 
The problem that could happen is with box2d when you need to create bodies, tiled/libgdx will not create bodies for you. When you create bodies and add them to box2d world. There is box2d renderer that will render body shapes and after map renderer would still render the objects that represent box2d bodies if your feature request gets implemented.
My suggestion would be, just get shapes from object layer and draw them with shape renderer, later when you don't need them anymore just remove drawing code for those shapes.

**Comment by nathanielbrown** [comment]
I agree that none of the other shapes should be rendered but I thought it would make sense for the TiledMapTileMapObject to be rendered or if there was a debug option that could render all the shapes just like the box2D debug render.

When I was setting up my game this is basically what I did. In the end I did exactly what you said. I took them information out of the layer and built my world with it.

I was mainly thinking of new developers that add Texture Objects and they don't understand why they cannot see them.
